ext/dom/characterdata.c

```diff
@@ -38,19 +38,13 @@ URL: http://www.w3.org/TR/2003/WD-DOM-Level-3-Core-20030226/DOM3-Core.html#core-
 int dom_characterdata_data_read(dom_object *obj, zval *retval)
```

Comment:
Complete side note, changing the return type from various functions that only return ``SUCCESS`` and ``FAILURE`` from ``int`` to ``zend_result`` might be a good idea as a follow-up.